### PR TITLE
Update kOps command for creating clusters

### PR DESCRIPTION
**Pull request by hakman**
**What type of PR is this?**
> Uncomment only one, leave it on its own line:
>
> /kind api-change
> /kind bug
> /kind cleanup
> /kind design
> /kind documentation

/kind failing-test

> /kind feature
> /kind flake

**What this PR does / why we need it**:

Fixes e2e test failure seen in https://github.com/kubernetes/cloud-provider-aws/pull/552.

**Which issue(s) this PR fixes**:

n/a

**Special notes for your reviewer**:
n/a

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: hack/e2e/run.sh [hunk 1]
```diff
@@ -128,7 +128,7 @@ if [[ "${UP}" = "yes" ]]; then
       --run-id="${test_run_id}" \
       --cloud-provider=aws \
       --cluster-name="${CLUSTER_NAME}" \
-      --create-args="--dns=none --zones=${ZONES} --node-size=m5.large --master-size=m5.large --override=cluster.spec.kubeAPIServer.cloudProvider=external --override=cluster.spec.kubeControllerManager.cloudProvider=external --override=cluster.spec.kubelet.cloudProvider=external --override=cluster.spec.cloudControllerManager.cloudProvider=aws --override=cluster.spec.cloudControllerManager.image=${IMAGE_NAME}:${IMAGE_TAG} --override=spec.cloudConfig.awsEBSCSIDriver.enabled=true" \
+      --create-args="--dns=none --zones=${ZONES} --node-size=m5.large --master-size=m5.large --override=cluster.spec.cloudControllerManager.cloudProvider=aws --override=cluster.spec.cloudControllerManager.image=${IMAGE_NAME}:${IMAGE_TAG}" \
```
Comment (kishorj) [inline hack/e2e/run.sh:131]: do we not need to explicitly override the  awsEBSCSIDriver anymore?

**Review by kishorj — APPROVED** [review]

File: hack/e2e/run.sh (hunk 1 above)
Comment (hakman) [reply hack/e2e/run.sh:131]: awsEBSCSIDriver is enabled by default these days: https://github.com/kubernetes/kops/commit/658223531234e6c98a4bd59483cd13da655e4057.

**Pull request merged**